web/src/features/panels/zone/ZoneHeaderTitle.tsx

```diff
@@ -31,49 +31,41 @@ export default function ZoneHeaderTitle({ zoneId }: ZoneHeaderTitleProps) {
   const onNavigateBack = () => setIsMapMoving(false);
 
   return (
-    <div className="flex w-full grow flex-row overflow-hidden py-2 pl-2">
+    <div className="flex w-full py-2 pl-2">
       <Link
         className="self-center py-4 pr-4 text-xl"
         to={returnToMapLink}
         data-test-id="left-panel-back-button"
         onClick={onNavigateBack}
       >
-        <HiArrowLeft />
+        <ArrowLeft />
       </Link>
 
       <div className="w-full overflow-hidden">
-        <div className="flex w-full  flex-row justify-between">
-          <div className="mb-0.5 flex  w-full  justify-between">
-            <div className="flex w-full flex-row items-center pr-4 sm:pr-0 ">
-              <CountryFlag
-                zoneId={zoneId}
-                size={18}
-                className="shadow-[0_0px_3px_rgba(0,0,0,0.2)]"
-              />
-              <TooltipWrapper
-                tooltipContent={showTooltip ? zoneNameFull : undefined}
-                side="bottom"
-              >
-                <div className="ml-2 flex w-full flex-row overflow-hidden">
-                  <h1 className="truncate font-medium" data-test-id="zone-name">
-                    {zoneName}
-                  </h1>
-                  {showCountryPill && (
-                    <div className="ml-2 flex w-auto items-center rounded-full bg-gray-200 px-2 py-0.5  text-sm dark:bg-gray-800/80">
-                      <p className="w-full truncate">{countryName ?? zoneId}</p>
-                    </div>
-                  )}
-                </div>
-              </TooltipWrapper>
-              {disclaimer && (
-                <TooltipWrapper side="bottom" tooltipContent={disclaimer}>
-                  <div className="ml-2 mr-4 h-6 w-6 shrink-0 select-none rounded-full bg-white text-center drop-shadow dark:border dark:border-gray-500 dark:bg-gray-900">
-                    <p>i</p>
-                  </div>
-                </TooltipWrapper>
-              )}
+        <div className="flex w-full items-center gap-2 pr-4 ">
+          <CountryFlag
+            zoneId={zoneId}
+            size={18}
+            className="shadow-[0_0px_3px_rgba(0,0,0,0.2)]"
+          />
+          <TooltipWrapper
+            tooltipContent={showTooltip ? zoneNameFull : undefined}
+            side="bottom"
+          >
+            <h1 className="grow truncate" data-test-id="zone-name">
+              {zoneName}
+            </h1>
+          </TooltipWrapper>
+          {showCountryPill && (
+            <div className="flex w-auto items-center rounded-full bg-gray-200 px-2 py-0.5 text-sm dark:bg-gray-800/80">
+              <p className="w-full truncate">{countryName ?? zoneId}</p>
             </div>
-          </div>
+          )}
```

Comment:
I think it makes more sense to keep the pill left aligned as it was before.

Before:
![image](https://github.com/user-attachments/assets/713cd1ea-3fa1-4947-8b47-5eac1eea9203)

Now:
![image](https://github.com/user-attachments/assets/b8446517-d640-411c-bdab-f424fce64e81)

Reply:
I can change it back but my thinking was that this reduces CLS when switching between US zones as it keeps the USA badge in the same place.

Do you think that it's important that it's left aligned?

Reply:
I don't think it is that important but I think it looks a bit strange in a case like this:

![image](https://github.com/user-attachments/assets/82d7a190-24ec-41b6-8768-c07266e1c618)

I was also wondering if it was intentional to remove padding from the pill? Maybe it's just me being used to how it was before but I think the pill could use a bit more padding.

Reply:
> I don't think it is that important but I think it looks a bit strange in a case like this:
> 
> ![image](https://private-user-images.githubusercontent.com/95029552/359120479-82d7a190-24ec-41b6-8768-c07266e1c618.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwNzUxMDEsIm5iZiI6MTcyNDA3NDgwMSwicGF0aCI6Ii85NTAyOTU1Mi8zNTkxMjA0NzktODJkN2ExOTAtMjRlYy00MWI2LTg3NjgtYzA3MjY2ZTFjNjE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE5VDEzNDAwMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMwY2RlZTEzOWM5NTljZDAyYTQ0NDM3ODc3YTAxZmFjNzgxOTg2NGVhZWQyY2ZkMTAxODhhNzE4YThkODU3MzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.G93ec2NMKZU1Qh2DOngj-KIL9yS_PoKxW04BzXZUsYk)
> 
> I was also wondering if it was intentional to remove padding from the pill? Maybe it's just me being used to how it was before but I think the pill could use a bit more padding.

I didn't change the padding on the pill 👀 only replaced the left margin with a gap for the entire group.

But I can change back the alignment and then we can reevaluate that later, the custom pill should probably be changed to use the Badge component anyhow.